source/app/service-providers/fsal/util/file-parser.ts

```diff
@@ -115,6 +115,21 @@ export default function getMarkdownFileParser (
           file.yamlTitle = title
         }
       }
+
+      const tagsFromFrontmatter = []
+      if (file.frontmatter.tags) {
+        tagsFromFrontmatter.push(...file.frontmatter.tags)
```

Comment:
This assumes that `file.frontmatter.tags` is an array, but it can be a string just as well, so you should add a few more checks. Have a look at the old implementation that worked well:

https://github.com/Zettlr/Zettlr/blob/v2.3.0/source/app/service-providers/fsal/util/extract-tags.ts